enterprise/backend/src/metabase_enterprise/scim/api.clj

```diff
@@ -2,17 +2,46 @@
   "/api/ee/scim/ endpoints"
   (:require
    [compojure.core :refer [POST]]
+   [metabase-enterprise.serialization.v2.backfill-ids :as serdes.backfill]
    [metabase.api.common :as api :refer [defendpoint]]
    [metabase.models.api-key :as api-key]
+   [metabase.models.setting :as setting :refer [defsetting]]
+   [metabase.public-settings :as public-settings]
+   [metabase.util.i18n :refer [deferred-tru]]
    [metabase.util.secret :as u.secret]
    [toucan2.core :as t2]))
 
 (set! *warn-on-reflection* true)
 
+(defsetting scim-enabled
+  (deferred-tru "Is SCIM currently enabled?")
+  :visibility :admin
+  :type       :boolean
+  :audit      :getter
+  :default    false
+  ;; TODO: should this be exported?
+  :export?    false)
+
+(defsetting scim-base-url
+  (deferred-tru "Base URL for SCIM endpoints")
+  :visibility :admin
+  :type       :string
+  :setter     :none
+  :audit      :never
+  :export?    false
+  :getter     (fn []
+                (str (public-settings/site-url) "/api/ee/scim/v2")))
```

Comment:
Asking because I'm curious: Is this implemented as a setting because that is a convenient way to have it get picked up and included in the properties json response?

Reply:
Basically yeah